python-backend/app/api/mines/status/models/mine_operation_status_code.py

```diff
@@ -22,23 +22,25 @@ def json(self):
         return {
             'mine_operation_status_code': self.mine_operation_status_code,
             'description': self.description,
-            'display_order': str(self.display_order),
-            'effective_date': self.effective_date.isoformat(),
-            'expiry_date': self.expiry_date.isoformat()
+            'display_order': str(self.display_order)
```

Comment:
Not in scope of these changes, but I'm not convinced that the backend should know about something like "display order" at all.

Reply:
Wouldn't this be for the front-end to consume and sort there? Alternative could be to have the backend return the collection sorted by display_order and then not include this property in the returned json. This is maybe better. Right now, I don't think we consume these at all as the get is still returned from the constants file.

Reply:
I think I agree.. My impression of the idea behind this attribute is to let the frontend know how to order the items. This seems silly. If the backend has an opinion on how things should be ordered then, like you said, it can just order them ahead of time and not expose this attribute.

Otherwise, (and this is my current stance on it) the frontend knows that it wants to display these items in an order based on a certain field and takes care of that on its own.

With the current approach, it seems like both frontend and backend are contributing to the "ordering," which I think is used in dropdowns? Again, out of scope, but I noticed this due to the diff.